src/tilingresult.h

```diff
@@ -23,6 +23,7 @@ class TilingResult {
 public:
     TilingResult() = default;
     TilingStep& operator[](Client* client);
```

Comment:
Remove this? I wonder why the build doesn't fail. Is it because it is never called? 🤔

Reply:
Right, thanks!
Yes, the build doesn't fail because nothing links to it.